modules/core/shared/src/main/scala/retry/RetryPolicies.scala

```diff
@@ -119,7 +128,10 @@ object RetryPolicies {
         case GiveUp => GiveUp
       }
 
-    RetryPolicy[M](decideNextRetry)
+    RetryPolicy.withShow[M](
+      decideNextRetry,
+      show"limitRetriesByDelay(threshold=$threshold, ${policy.show})"
```

Comment:
You don't need the `.show` here, right? Just `$policy`, as the policy has a `Show` instance.

Same goes for a few other places below.